app/actions/sortActions.js

```diff
@@ -0,0 +1,36 @@
+
+import types from 'constants/ActionTypes';
+
+import { CALL_API } from 'redux-api-middleware';
+
+import schemas from 'store/middleware/Schemas';
+import {
+  buildAPIUrl,
+  getErrorTypeDescriptor,
+  getHeadersCreator,
+  getRequestTypeDescriptor,
+  getSuccessTypeDescriptor,
+} from 'utils/apiUtils';
+
+function sortResources(orderBy) {
+  return {
+    [CALL_API]: {
```

Comment:
Change CALL_API to RSAA, you're using `very old` API, remove your `node_modules` and do `yarn install` again @Rasbin

Reply:
Actually, found that this file re-appeared somehow which is not in use. So, I removed it in latest commit.